### PR TITLE
Fix unquoting

### PR DESCRIPTION
**Pull request by lucasts**
use yarl public api to unquote

### CODE REVIEW

File: aiohttp_devtools/runserver/serve.py [hunk 1]
```diff
@@ -295,7 +295,7 @@ def modify_request(self, request):
         """
         Apply common path conventions eg. / > /index.html, /foobar > /foobar.html
         """
-        filename = unquote(request.match_info['filename'])
+        filename = URL(request.match_info['filename'], encoded=True).path
```
Comment (asvetlov) [inline aiohttp_devtools/runserver/serve.py]: `URL.build(path=...)` is even better.

**Comment by lucasts** [comment]
updated travis.yml as well to test against current aiohttp version

**Comment by deterok** [comment]
Please, merge it :)

File: aiohttp_devtools/runserver/serve.py (hunk 1 above)
Comment (lucasts) [reply aiohttp_devtools/runserver/serve.py]: Done.

Comment (asvetlov) [reply aiohttp_devtools/runserver/serve.py]: Looks like not :)

**Comment by samuelcolvin** [comment]
please fix and I'll merge

**Comment by lucasts** [comment]
not sure what to do with the failing aiohttp 2.0 test(using yarl 0.10.3)
any directions @samuelcolvin / @asvetlov ?

**Comment by samuelcolvin** [comment]
as per #164 I think we're happy to drop `2.0.0`, shall we support only `>2.3.0`?

**Comment by samuelcolvin** [comment]
I'm replacing this with #170 as the changed required conflicted and I want to release an update.

**Pull request closed without merging**